io/zenoh-transport/src/unicast/establishment/accept.rs

```diff
@@ -637,16 +639,19 @@ impl<'a, 'b: 'a> AcceptFsm for &'a mut AcceptLink<'b> {
 }
 
 pub(crate) async fn accept_link(link: LinkUnicast, manager: &TransportManager) -> ZResult<()> {
+    let direction = TransportLinkUnicastDirection::Inbound;
     let mtu = link.get_mtu();
     let is_streamed = link.is_streamed();
     let config = TransportLinkUnicastConfig {
-        direction: TransportLinkUnicastDirection::Inbound,
+        direction,
         batch: BatchConfig {
             mtu,
             is_streamed,
             #[cfg(feature = "transport_compression")]
             is_compression: false,
         },
+        reliability: link_reliability(&link),
+        priority: link_priority(&link, direction)?,
```

Comment:
I believe this won't work since I'm not sure the metadata are correctly set/stored in the link.

Reply:
I'm assuming this comment is referring to the next comment.

Reply:
Priority ranges are not obtained from `INIT` messages using the `QoS` extension.